discovery_engine_core/core/Cargo.toml

```diff
@@ -39,11 +39,10 @@ sqlx = { version = "0.6.0", features = ["runtime-tokio-rustls", "sqlite", "uuid"
 
 [dev-dependencies]
 async-once-cell = "0.4.2"
-claim = "0.5.0"
```

Comment:
for whatever reason `claim` fixes a subdependency to an old minor version, which results in version conflicts for example with `tokio`. i removed `claim` and replaced its macros with `std` macros.